build/resources/main/META-INF/mods.toml

```diff
@@ -1,63 +1,61 @@
 modLoader = "javafml"
-loaderVersion = "[43,)"
+loaderVersion = "[47,)"
 license = "MIT"
 [[mods]]
     modId = "kubejsminestuck"
     version = "${file.jarVersion}"
     displayName = "KubeJSMinestuck"
     logoFile="wtd.png"
-    authors="Love, Cheese and small house plants"
-    description = '''
-KubeJS support for Minestuck Recipes and Resources
-'''
+    authors="hadean"
+    description = "Adds Minestuck support for KubeJS, including recipes, custom grist types, and sburb data."
 
 [[dependencies.kubejsminestuck]]
-modId="minestuck"
-mandatory=true
-versionRange="[1.9.0,)"
-ordering="NONE"
-side="BOTH"
-[[dependencies.kubejsminestuck]]
-modId="geckolib3"
-mandatory=true
-versionRange="[3.0,)"
-ordering="NONE"
-side="BOTH"
-[[dependencies.kubejsminestuck]]
-modId="infiniverse"
-mandatory=true
-versionRange="[1.0.0,)"
-ordering="NONE"
-side="BOTH"
+    modId="minestuck"
+    mandatory=true
+    versionRange="[1.11.0,)"
```

Comment:
maybe change this to match the version used in build.gradle?

Reply:
I fixed it under the other mods.toml file, this one is auto generated when it builds